tuna/metadata.py

```diff
@@ -279,7 +281,7 @@
 FUSION_COLS = [v for _, v in TABLE_COLS_FUSION_MAP.items()]
 CONV_COLS = [v for _, v in TABLE_COLS_CONV_MAP.items()]
 
-CONV_SKIP_ARGS = ['i', 't', 'V', 's', 'b', 'w', 'S']
+CONV_SKIP_ARGS = ['i', 't', 'V', 's', 'b', 'w', 'S', 'Z']
```

Comment:
what is Z used for?

Reply:
``` -Z        tensor vectorization type (none, vect_c, vect_n) (Default=0)```
not sure how to incorporate this

Reply:
Let see if @JehandadKhan  has some more insight. Is this something we need to address in a different way in Tuna?